setup.cfg

```diff
@@ -31,6 +31,7 @@ console_scripts =
    pbench-list-tools = pbench.cli.agent.commands.tools.list:main
    pbench-list-triggers = pbench.cli.agent.commands.triggers.list:main
    pbench-register-tool-trigger = pbench.cli.agent.commands.triggers.register:main
+   pbench-results-push = pbench.cli.agent.commands.results:results_push
```

Comment:
So you're defining the new `pbench-results-push` command but it's not actually being used anywhere? Is this an oversight, or an intentional separate PR?

Reply:
None of the above, as such.

The intention is to modify the existing `pbench-move-results`/`pbench-copy-results` script to use the new `pbench-results-push` tool.  However, in order for that to be viable, we'll need a server with the PUT API deployed for it to target.

In the meantime, the `pbench-results-push` tool can/will be used to test-load existing (archived) tarballs into the new RDU2 server.